testdata/globalconfig.yml

```diff
@@ -6,12 +6,8 @@ adapters:
     impl: memQuota
     params:
   - name: default
-    kind: access-logs
```

Comment:
Yes. `kind` is specified by the adapter implementation now. It is no longer needed  (allowed) in the global config.